### DISCUSSION TITLE
Use GitHub for Rectangle Pro releases

### DISCUSSION THREAD

**Discussion by quinncomendant** [rxhanson/RectanglePro-Community#457] [General]
I noticed you have been uploading new Rectangle Pro releases [here](https://github.com/rxhanson/Rectangle/releases), including a changelog. It would be helpful if the releases were posted to GitHub so I can subscribe to receive notifications for new versions and see what changed, similar to show for [Rectangle releases](https://github.com/rxhanson/Rectangle/releases).

I know Rectangle Pro is not open source, but you can still use GitHub releases for the binary files and changelog. 

Just an idea. :)

Thanks for such cool software!

**Comment by rxhanson** [comment]
Thanks for the feedback. Although this could likely be automated to some degree, it's quite nice to just deploy to just one place. Also, I kinda feel like I would be taking advantage of this by using it for file hosting as well as my issue tracker for a closed source app.
